### PR TITLE
MaxPool-8: pads_value attribute removal from the operator definition

### PR DESCRIPTION
**Pull request by tomdol**
Adjusting the op to match latest specification changes https://github.com/openvinotoolkit/openvino/pull/5359

### CODE REVIEW

**Review by ilyachur — APPROVED** [review]
Should we wait for specification merge?

**Review by lazarevevgeny — APPROVED** [review]

**Comment by lazarevevgeny** [comment]
> Should we wait for specification merge?

I think that we can merge now to unblock anyone who is waiting for this op.

**Comment by tomdol** [comment]
The spec is almost ready to be merged too, just 2 remaining CI checks are currently in progress. The final version of the operator has been agreed and the spec is updated accordingly.

**Pull request merged**
